CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Contributing to Grafana's Shared Workflows Repository
+
+Welcome to the Shared Workflows repository! This repository contains reusable actions designed to streamline and automate common tasks across multiple departments and external organizations. We appreciate your interest in contributing to our shared efforts.
+
+## Scope of `shared-workflows`
+
+This repository is for generic workflows and actions which can be used across multiple types of projects. For more specialised use-cases, we encourage the use of project- or area-specific shared workflow repositories.
+Examples which would fit in here:
+
+- Push a container image to a Docker registry
+- Run security scanners on built artifacts
+- Read and write objects to cloud object storage (GCS or S3)
```

Comment:
...and the generic one can go here

```suggestion
- Read and write objects to cloud object storage (GCS or S3)
- Lint projects with a single (configurable) toolset
```

(I removed the "(s)" since we don't do that elsewhere.)

although, to be honest, I find that wording a bit unclear. I'm not sure we would simply wrap a linter here, since we don't have global coding standards for the `grafana` org (AFAIK). So we would end up exposing all/most of the options from the underlying action, in which case projects might as well use it directly.

So maybe we can leave this one out. Or, since one action we have is "lint PR title", we could say something like

```
- Lint projects according to a widely-agreed specification such as [Commitlint](https://commitlint.js.org/).
```

WDYT? also cc @zerok

Reply:
I like the mention of `widely-agreed specification`... I think it represents the point we want to drive home. 

Added and pushed.